documentation/docs/audit_trail_gw_api_reference.md

```diff
@@ -0,0 +1,789 @@
+# Audit Trail GW API Documentation
+This is the API documentation for the Audit Trail Gateway of the [Integration Services](https://github.com/iotaledger/integration-services). For further information have a look at our [Wiki](https://wiki.iota.org/integration-services/services/audit-trail-gateway/introduction).
+
+## Version: v0.1
```

Comment:
I don't know how customizable this is since I didn't write the generation but we can put some time into researching it. Please create a ticket at the IS describing what you would have like.